plugins/powerd/fu-plugin-powerd.c

```diff
@@ -20,6 +20,40 @@ fu_plugin_init(FuPlugin *plugin)
 	fu_plugin_alloc_data(plugin, sizeof(FuPluginData));
 }
 
+static gboolean
+fu_plugin_powerd_create_suspend_file(GError **error)
+{
+	g_autofree gchar *lockdir = NULL;
+	g_autofree gchar *inhibitsuspend_filename = NULL;
+	g_autofree gchar *getpid_str = NULL;
+        gboolean file_present = FALSE;
+
+	lockdir = fu_common_get_path(FU_PATH_KIND_LOCKDIR);
+	inhibitsuspend_filename = g_build_filename(lockdir, "power_override", "fwupd.lock", NULL);
+	getpid_str = g_strdup_printf("%d", getpid());
+	file_present = g_file_set_contents(inhibitsuspend_filename, getpid_str, -1, error);
+        if (!file_present)
```

Comment:
Wrong formatting... Lets get the `contrib/setup` figured out so this stops happening.